src/backend/c-lisp.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env python3
+import json
+import sys
+import utils.utils
+
+CLISP_PREFIX = "tmp_clisp"
+
+
+def random_label(prefix=CLISP_PREFIX, extra_prefixes=[], length=10):
+    return utils.utils.random_label(prefix, extra_prefixes, length)
```

Comment:
utils.utils is not nice at all. I like the previous namespacing.

Reply:
Done